src/Controller/Api/Coalition/CauseAnonymousFollowerController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Controller\Api\Coalition;
+
+use App\Entity\Coalition\Cause;
+use App\Entity\Coalition\CauseFollower;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * @Route("/causes/{uuid}/follower", name="api_follow_cause_as_anonymous", methods={"PUT"})
+ */
+class CauseAnonymousFollowerController extends AbstractController
+{
+    public function __invoke(
+        Request $request,
+        Cause $cause,
+        SerializerInterface $serializer,
+        ValidatorInterface $validator,
+        EntityManagerInterface $entityManager
+    ): Response {
+        /** @var CauseFollower $causeFollower */
+        $causeFollower = $serializer->deserialize($request->getContent(), CauseFollower::class, JsonEncoder::FORMAT);
+        $causeFollower->setCause($cause);
+
+        $errors = $validator->validate($causeFollower, null, ['write']);
```

Comment:
why do we need to use a custom validation group `write`, can we use `Default`?

Reply:
Because we should have 2 different validations for follow by adherent and by anonymous.

Reply:
oh I see, in this case we should validate two groups: `['Default', 'anonymous_follower']`, I suggest to rename `write` group to `anonymous_follower`, wdyt?
`CauseFollower::cause` for exemple must have `@NotBlank` constraint with `Default` validation group.